testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -13063,3 +13065,425 @@ fn tenure_extend_cost_threshold() {
 
     signer_test.shutdown();
 }
+
+#[test]
+#[ignore]
+/// Test that a miner that begins mining before seeing the last block of the
+/// previous tenure can be interrupted when its tip advances to the last block,
+/// then successfully mine a block on top of that block.
+///
+/// Test Setup:
+/// The test spins up five stacks signers, one miner Nakamoto node, and a corresponding bitcoind.
+/// The stacks node is then advanced to Epoch 3.0 boundary to allow block signing.
+///
+/// Test Execution:
+/// The node mines 1 stacks block N (all signers sign it). The subsequent block N+1 is proposed, but <30% accept it. The remaining signers
+/// do not make a decision on the block. A new tenure begins and the miner proposes a new block N+1' which all signers accept.
+///
+/// Test Assertion:
+/// Stacks tip advances to N+1'
```

Comment:
I think this rustdoc isn't accurate

Reply:
Fixed in 5308a99778e7b0155f74691abb9eb0abf63b8b93.